Description: Make: add --as-needed ld flag
  Avoids unnecessary linking, as suggested by Eugene Paskevich
Author: Tias Guns <tias@ulyssis.org>
Applied-Upstream: commit:fb22ed1a877444e2c970, http://github.com/tias/xinput_calibrator/commit/fb22ed1a877444e2c97031a00066d411e2d228ee

--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -43,6 +43,10 @@
 xinput_calibrator_SOURCES = main_gtkmm.cpp
 xinput_calibrator_LDADD = $(XINPUT_LIBS) $(GTKMM_LIBS)
 xinput_calibrator_CXXFLAGS = $(XINPUT_CFLAGS) $(GTKMM_CFLAGS) $(AM_CXXFLAGS)
+
+# only include the needed gtkmm stuff
+# lets hope this has no side-effects
+xinput_calibrator_LDFLAGS = -Wl,--as-needed
 endif
 
 EXTRA_DIST = \
